Server/src/main/java/local/ModelDAOImpl.java

```diff
@@ -32,7 +32,7 @@
      * @author Paul
      */
     @Override
-    public String getModelNameFromAssetTypeID(String assetTypeID) {
+    public String getModelNameFromModelID(String assetTypeID) {
```

Comment:
It would be good to update the parameter name and description of the method at the same time.